src/index.js

```diff
@@ -58,7 +62,8 @@
 
     return {
       document: domBuilder.fragment.node,
```

Comment:
This is causing the main issues with live editor, an entirely different document is being constructed and returned.

Reply:
There were [previous Slowparse changes](https://github.com/mozilla/slowparse/commit/4638878e0ab9ad85f26aaffb851fc0cd6a223c78) that introduced this new document construction, but this piece of code is the most relevant.